assets/js/pages/AnalyticsEula/AnalyticsEulaModal.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+
+import Modal from '@common/Modal';
+import Button from '@common/Button';
+import Input from '@common/Input';
+
+function AnalyticsEulaModal({
+  isOpen = false,
+  checked = false,
+  onChecked,
+  onEnable,
+  onCancel,
+}) {
+  return (
+    <Modal
+      className="!w-1/2"
+      title="Collection of Anonymous Metrics"
+      open={isOpen}
+      onClose={() => null}
+    >
+      Allow the collection of{' '}
+      <a
+        className="text-jungle-green-500 hover:opacity-75"
```

Comment:
question:
I guess we will need to have a proper page on analytics, right?

Reply:
Yes, we don't have a proper Analytics page in the docs yet. This will be added as a separate task. I also noticed the url was broken so I fixed it.